ceremonies/src/lib.rs

```diff
@@ -427,7 +427,7 @@ pub mod pallet {
 			Ok(().into())
 		}
 
-		#[pallet::weight((1000, DispatchClass::Operational,))]
+		#[pallet::weight((<T as Config>::WeightInfo::set_meetup_time_offset(), DispatchClass::Normal, Pays::Yes))]
```

Comment:
don't know how criticsal this is, but such errors could be detected by some custom check script we run in CI to ensure no hardcoded weights are present

Reply:
Yeah makes sense. `1000` is very little and would make an attack quite cheap.